### PR TITLE
Fixed gem file problems

### PR DESCRIPTION
**Pull request by allenzhong**
Fixed gem file problems when team merged

### CODE REVIEW

**Comment by yuxiang8931** [comment]
Ok

**Comment by JohnnyJ0622** [comment]
It's ok

File: Gemfile [hunk 1]
```diff
@@ -43,9 +42,6 @@ gem 'cancancan'
 gem 'devise'
 
 group :development, :test do
```
Comment (yuxiang8931) [inline Gemfile]: There are two test groups new.

File: Gemfile [hunk 2]
```diff
@@ -57,9 +53,18 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
-  gem 'cancancan'
```
Comment (yuxiang8931) [inline Gemfile]: Where did you put 'cancancan' then?

Comment (allenzhong) [inline Gemfile]: at line 41.

File: Gemfile (hunk 1 above)
Comment (allenzhong) [inline Gemfile]: for common gem that should be used in both development and test, it should be put into  `group :development, :test do`. For gems only using in test such as `cucumber`, `rspec` and etc.,  it just should be put into `group :test do` block.

File: Gemfile (hunk 2 above)
Comment (yuxiang8931) [inline Gemfile]: No problem

File: Gemfile (hunk 1 above)
Comment (yuxiang8931) [inline Gemfile]: OK

**Comment by JohnnyJ0622** [comment]
ok

**Pull request merged**